src/Behat/Behat/EventDispatcher/Event/AfterScenarioSetup.php

```diff
@@ -65,9 +65,7 @@ public function getFeature()
     }
 
     /**
-     * Returns scenario node.
-     *
-     * @return ScenarioNode
+     * @return Scenario
```

Comment:
annotation was wrong and I used the opportunity a comment which does not provide any useful information